### ISSUE TITLE
pkgfile useless due to lack of .files in repos

### ISSUE THREAD

**Issue by tgharib** [manjaro/boxit#3]
Discussion: https://forum.manjaro.org/index.php?topic=9268.0

**Comment by hphilm** [comment]
Yes, I know of that. We won't support it, though ...

**Comment by tgharib** [comment]
@philmmanjaro That's slightly disappointing. The reason why I need it is because of [debtap](https://github.com/helixarch/debtap). Every now and then, there's a package (like uberwriter) that's not in the arch/manjaro repos nor is it in the AUR so I need to use a DEB file. I actually switched from Manjaro to vanilla Archlinux for this functionality but I'm probably going to switch back to Manjaro in the future and just keep a VM of Archlinux when I need to convert a package. Slightly annoying but I guess your dev time is better focused on things like pamac.

Of course you'll find uberwriter in the AUR now but that's only because I created an AUR package after using debtap to assist me in converting the required dependencies.
